packages/tpetra/core/src/Tpetra_CrsMatrix_def.hpp

```diff
@@ -5927,42 +5927,36 @@ namespace Tpetra {
         Distributor& dist) const
   {
```

Comment:
This is the pack() method.  We do have the option to change that method's interface, for example to take `Kokkos::View` or `Kokkos::DualView` instead of `Teuchos::Array*`.  We may want to consider doing that before we find ourselves copying in and out a lot.